.gitlab-ci.yml

```diff
@@ -328,6 +328,16 @@ pinpoint-check:
     - *yarn_install
     - make lint_country_dialing_codes
 
+audit_packages:
```

Comment:
```suggestion
audit_packages:
  needs:
    - job: install
```

Reply:
How come `pinpoint-check` doesn't need that?

Reply:
We have jobs in the `test` stage that don't require the Ruby/JavaScript dependencies, so we've added the above `needs` block to the ones that do to ensure dependency caching can be used.

Reply:
We should probably add it in `pinpoint-check` as well.

Reply:
ok! added for `pinpoint-check` as well in 39294977dbcc7109c120dc51187f949e088b3f3e